core/tee/tee_svc.c

```diff
@@ -43,6 +42,12 @@
 #include <trace.h>
 #include <kernel/trace_ta.h>
 #include <kernel/chip_services.h>
+#include <kernel/misc.h>
+
+#define MAX_BASE64_BUFFER_SIZE 120
+
+extern int base64_encode(const unsigned char *in,  unsigned long len,
```

Comment:
Ah, yes, you're right. I missed this from the specification:
> (TEE_GetPropertyAsBinaryBlock) Otherwise, if this function succeeds, then calling the function TEE_GetPropertyAsString on the same name and with a sufficiently large output buffer MUST also succeed and return a string that is consistent with a Base64 encoding of the binary block [...]

Then, I think it makes sense to add base64_encode to the crypto_ops structure.